Tests/Test1/ThreadSafePriorityQueue/ThreadSafePriorityQueue.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace ThreadSafePriorityQueue;
+
+/// <summary>Thread-safe priority queue.</summary>
+/// <typeparam name="TV">The type of elements.</typeparam>
+/// <typeparam name="TP">The type of priority. Inheritance from <see cref="IComparable"/> required.</typeparam>
+public class ThreadSafePriorityQueue<TV, TP>
```

Comment:
```suggestion
public class ThreadSafePriorityQueue<Value, Priority>
```
Имена типовых переменных - это имена. Поэтому они должны быть содержательными.